nl_server/routes.py

```diff
@@ -21,6 +21,41 @@
 bp = Blueprint('main', __name__, url_prefix='/')
 
 
+@bp.route('/api/embedding/index/', methods=['GET'])
```

Comment:
what is the index here? Can we use '/api/embedding?index=' as the endpoint, compared with '/api/embedding?q='

Reply:
I have combined the two endpoints in to one. "index" is when you want to get the embeddings at a specific index in the mode. query string is used when we want to get a new embeddings vector for any string.